lab_4_fill_words_by_ngrams/main.py

```diff
@@ -39,6 +61,15 @@ def _put(self, element: str) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(element, str) or not element:
+            raise ValueError
+
+        if element in self._storage:
+            return None
+
+        self._storage[element] = len(self._storage)
+
+        return None
```

Comment:
it is not beautiful

Reply:
ok, but i just copypasted it from the reference code of the _put method in the LAB3.. just thought it would be kind of correct to do that
![image](https://github.com/fipl-hse/2023-2-level-labs/assets/66551692/d9ddfd1f-a90b-4f77-94eb-cae52eb35011)

Reply:
and the last return is ugly

Reply:
already fixed it